pkg/logger/logger.go

```diff
@@ -188,6 +188,25 @@ func isAggregateSetAndIsLogNotNew(skip int, l *Logger) bool {
 
 // Log functions
 
+// BPFError logs errors from bpf programs
+// It does not use the aggregation logic as the bpf errors are already aggregated on the bpf side
+func BPFError(msg string, logLevel Level, keysAndValues ...interface{}) {
```

Comment:
I'd keep this function near `processBPFErrors` rather than here, since it's more of a readability helper rather than something a consumer of the logger package would call from anywhere else.

Reply:
This is proposed inside the logger package because the default behaviour of this package is to aggregate calls if logger aggregation is on. For the case of the ebpf error coming from the perfbuf it doesn't make sense, so we need a logger helper that skips the inner aggregation. Its naming `logger.BPFError()` can be changed for sure. WDYT?

Reply:
As far as I see all this does is switch on the log level and call the appropriate public logger function, with a certain field enrichment on the default case.
No aggregation logic is done in this helper function so I'm not sure what I'm missing.

Does the location of the function affect it's behavior?

Reply:
> Does the location of the function affect it's behavior?

Yep. Inside the pkg logger we can avoid the inner aggregation. Calling it from outside if the pkg aggregation is on, it will only aggregate.

See that we're bypassing the default behaviour calling it directly: `pkgLogger.l.Debugw(msg, keysAndValues...)`. If we call it from outside we'll face the pkg logger aggregation:

![image](https://user-images.githubusercontent.com/3372117/207149888-f2f52872-0c41-4c09-b61d-fd24a3d01ea9.png)

Besides that, the default behaviour of the debugw is append the origin from the user space debug call. We don't want that for ebpf errors.

Reply:
Regarding this, if necessary we can change the `logger.BPFError()` to something more generic like `logger.Log(aggregate=false, ...)` or rethink places for that. But for this release, I assume we're ready.

Reply:
Anything related to this, please let's take to https://github.com/aquasecurity/tracee/issues/2479.